### ISSUE TITLE
Prod docker images hardcoded to non-prod base images

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#902]
I just noticed that three of our docker images are incorrectly referencing non-production base images:

- [x] **cluster-manager**
- [x] **test-cadence**
- [x] **test-temporal**

The problem is that the **Dockerfile** files for these images hardcode the **nkubedev** Docker Hub organization.  This means that the production images built for **nkubeio** will be built from the development base image rather than the production base image.

Remember that we want development Neon images are pushed to **nkubedev** and production images to **nkubeio**.    The idea here is to allow potentially breaking development images to be isolated from production deployments as well as to keep the prod organization clean by not including tons debug images from potentially lots of different GIT branches.

The way we're accomplishing this is by passing an **ORGANIZATION** argument to the Dockerfile and then passing the correct Docker organization in the image's `build.ps1` script.  You can obtain this by calling the `DockerOrg` function.  This returns **nkubeio** if the current branch name begins with **release-** or **nkubedev** otherwise.

**Comment by jefflill** [comment]
I'm going to go ahead and fix these cases, but I wanted you guys to be aware of this so you can do the right thing in the future.

**Comment by jefflill** [comment]
I also just noticed that the **neon-log-host** `Dockerfile` references `${ORGANIZATION}/fluent-bit:${BRANCH}-latest` as its base image and there is a **nkubedev/fluentbit** image but there's no **nkubeio/fluentbit** image which means that the production image build will probably fail.

I'm also not sure where the **nkubedev/fluentbit** came from; there's no image definition for this in the GIT repo.  I suspect that this was manually pushed.  Should the `Dockerfile` be referencing somebody else's base image.

One of our goals is to be able to completely rebuild all of our images from scratch via the global `$/Images/publish.ps1` script.

**Comment by jefflill** [comment]
- [ ] **cluster-manager** service is a .NET Core console app but it's `Dockerfile` references the **aspnet** base image rather than the **dotnet** image.  Non APSNET services should use the **dotnet** base image; it's smaller and helps make it clear what the actual service requirements are.

**Comment by jefflill** [comment]
- [ ] Use **aspnet:latest** and **dotnet:latest** images?

I think we should use the latest base neon .NET base images rather than using explicit tags.  The **cluster-manager** `Dockerfile` currently looks like:
```
ARG         ORGANIZATION
FROM        ${ORGANIZATION}/aspnet:master-3.1.0-bionic
MAINTAINER  marcus@loopie.io
STOPSIGNAL  SIGTERM
ARG         APPNAME
```
There are a couple problems with this.  First, this includes the **master** branch tag.  The production **nkubeio** organization will never have images tagged with

**Comment by jefflill** [comment]
Closing because this will be addressed in #1110
